calcom/cal.com
# fix: Event Type Rescheduling disabled for events that are already cancelled

## What does this PR do?
This PR aims to disable rescheduling of cancelled events.
---
Fixes #11669 
---
Detailed Description
---
- Included  booking status from database in the queries
- If status is cancelled, redirect the user to cancelled booking page.
---
Before Changes 
Loom Video: https://www.loom.com/share/b88a4881f1cf4002bba0f6faab8f3ff0?sid=e52ad725-c438-4273-a5f3-5cf69cdbf4e7

## After Changes 
 Loom Video:  https://www.loom.com/share/0554d0110f06467498d839465f8f90a3?sid=8aa061e4-d6d4-459f-8053-e339484b36f7


## Type of change
## How should this be tested?
- Create a booking
- Go to the booking page link where we see option of rescheduling or cancel
- Copy the link to reschedule in new browser tab
- Now cancel the event and reschedule the event
- event successfully rescheduled but ideally it should not be.
- This pr fixes the above bug.

## Mandatory Tasks
